modules/nf-core/samtools/sormadup/meta.yml

```diff
@@ -0,0 +1,107 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "samtools_sormadup"
+description: Collate/Fixmate/Sort/Markdup SAM/BAM/CRAM file
+keywords:
+  - sort
```

Comment:
Perhaps add the rest of the tool subcommands to the keyword list

Reply:
Done!